components/Banner.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Banner = () => {
+  return (
+    <div>
+      <div className="banner">
+        <div className="stripe"></div>
```

Comment:
It is recommended to self close tags that have no children. Like this div here. Change to `<div className="stripe" />`

Read more [HERE](https://github.com/airbnb/javascript/tree/master/react#tags)

Reply:
Done